src/react/utils/__tests__/hooks.test.jsx

```diff
@@ -0,0 +1,72 @@
+import * as hooks from '../hooks';
+import React, { useRef } from 'react';
+import { useHeight, useOutsideClick } from '../hooks';
+import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
+
+const originalOffsetHeight = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetHeight');
+const originalOffsetWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetWidth');
+
+const Component = () => {
+    const ref = useRef(null);
+    const height = useHeight(ref);
+
+    useOutsideClick(ref, () => {
```

Comment:
Should not we check that the `useOutsideClick` callback is being called when we click outside of the component?
I did not understand how you test it.

Reply:
done: I was listening events from the document but I change that I now watch if useOutsideClick is triggered